AudioTranscriber.py

```diff
@@ -82,7 +85,9 @@ def process_speaker_data(self, data):
         return temp_file
 
     def get_transcription(self, file_path):
-        result = self.audio_model.transcribe(file_path, fp16=torch.cuda.is_available())
+        
+        result = self.audio_model.transcribe(file_path, fp16=torch.cuda.is_available(),task="translate")
```

Comment:
We are not passing in the language which we want to transcribe. task="translate"  will translate the speech into English. which is not the flag we want